app/frontend/src/registry/components/people/ApplicationAdd.vue

```diff
@@ -1,218 +1,156 @@
-              <input type="text" id="certNumber" placeholder="Enter certificate number" v-model="editClassification.certNo">
-            </div>
-          </div>
-          <div class="form-group">
-            <h4>Qualified to drill under this classification</h4>
-          </div>
-          <div class="form-group">
-            <div class="col-xs-12 col-sm-4 col-md-3">
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.wat"> Water supply wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.mon"> Monitoring wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.rech"> Recharge wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.inj"> Injection wells
-                </label>
-              </div>
+  <div class="card w-100">
+    <div class="card-body">
+      <h5 class="card-title">Classification, Qualifications &amp; Adjudication
+        <button type="button" class="close pull-right" aria-label="Close" v-on:click="$emit('close')">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </h5>
+      <p v-if="loading" class="card-text">
+        <b-row>
+          <b-col md="12">
+            <div class="fa-2x text-center">
+              <i class="fa fa-circle-o-notch fa-spin"></i>
             </div>
-            <div class="col-xs-12 col-sm-4 col-md-3 registry-item">
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.dewat"> Dewatering wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.rem"> Remediation wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.geo"> Geotechnical wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.clos"> Closed-loop geoexchange wells
-                </label>
-              </div>
-            </div>
-          </div>
-        </form>
-      </div>
+          </b-col>
+        </b-row>
+      </p>
+      <p v-else class="card-text">
+        <b-form-group label="Certification">
+          <b-row>
+            <b-col md="2">Issued by</b-col>
```

Comment:
We should include the `label-for="the-form-field-id"` attribute on form-groups so bootstrap-vue can generate the aria/screenreader attributes. I also think this works best with one form input per form group but I could be mistaken- try running it through Lighthouse

Reply:
Form-group documentation is confusing, it does say it's "purpose is to pair form controls with a legend or label". But it doesn't say how would have label-for apply to multiple form controls. I've changed the HTML to just have one label per input.